src/ts/game.ts

```diff
@@ -16,6 +16,8 @@ import { vecFromAngleAndScale } from './vector';
 const MAX_JUMP_CHARGE = 100;
 const JUMP_CHARGE_CYCLE_TIME_MS = 1000;
 const JUMP_CHARGE_RATE = (MAX_JUMP_CHARGE * 2) / JUMP_CHARGE_CYCLE_TIME_MS;
+const GRAVITATIONAL_CONSTANT = 0.00004;
```

Comment:
Why are you changing the value at the same time? I thought this was about displaying text.  Have you thoroughly tested changing the constant?

Reply:
Truth is, it was hard to get through some of the levels with the constants as they were before. This change makes the gravity a bit stronger, and increases the decay, so the planets that are far away have less influence (but not none). Just made it slightly more playable.